noxfile.py

```diff
@@ -224,6 +232,63 @@ def typing(session: nox.Session):
     )
 
 
+@nox.session
+@nox.parametrize(
+    ["major", "version", "extra_args"],
+    [
+        ["8", "8.0.0a3", []],
```

Comment:
I would switch to 8.0.0 instead of 8.0.0a3.

Reply:
Yeah. I can regenerate the data later.